gradle-jdks/src/main/java/com/palantir/gradle/jdks/GradleJdksConfigs.java

```diff
@@ -43,6 +45,15 @@ public abstract class GradleJdksConfigs extends DefaultTask {
     public static final String GRADLE_JDKS_SETUP_SCRIPT = "gradle-jdks-setup.sh";
     public static final String GRADLE_JDKS_FUNCTIONS_SCRIPT = "gradle-jdks-functions.sh";
 
+    /**
+     * Option to include a specific java major version when generating/checking the gradle jdk configuration files.
+     */
+    @Input
+    @Option(
+            option = "includeJava",
+            description = "Generates the jdk configuration directories for the Java major version.")
+    public abstract SetProperty<String> getIncludedJavaMajorVersion();
```

Comment:
I think we could avoid this if we were to regenerate the jdk configs at the same time (ie same gradle invocation in excavator) as bumping the versions? The problem should only present itself when changing the versions used, then trying to run gradle again but the wrong JDKs are included. I think we might be able to do that.

If that doesn't work, I think I'd prefer an option to just use all the configured versions rather than including specific ones. That way we don't need to keep updating excavators with specific versions, they can just include everything they need.

Reply:
I wonder if this should just be something you can set via an environment variable instead, so we can just set `export GRADLE_JDKS_INSTALL_ALL_JDKS=true` in the excavator and not have to worry about it setting it on all the different commands.